container/run.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env
```

Comment:
maybe im not understanding the flow correctly.
but this file runs when you first build the docker image, from the `Dockerfile` right?

but this file isn't doing anything? its just setting up functions and exports that's not being used. or am i missing something?

why not run those functions on the `else` case like in the other `run.sh` files? that way we dont have to attach to the container to run the app.

so we can just run `keg consumer start` && `keg consumer sync evf:install:start`

my initial expectation is that I can run `keg consumer start --sync evf:install:start` - but this is currently throwing errors. so maybe the first approach is fine

Reply:
@daniel112 - For this command `keg consumer start --sync evf:install:start`; that should work as you expect, but it's broken for injected apps. I haven't had a chance to look at it. I'm making a note to take a look when I have a chance

Reply:
Updated this to start the consumer when the container starts, consistent with our other containers